lib/record_store/provider/ns1.rb

```diff
@@ -12,7 +12,7 @@ def client
       # Downloads all the records from the provider.
       #
       # Returns: an array of `Record` for each record in the provider's zone
-      def retrieve_current_records(zone:, stdout: $stdout) # rubocop:disable Lint/UnusedMethodArgument
+      def retrieve_current_records(zone:, stdout: $stdout)
```

Comment:
Wait I'm confused, it's still not using stdout?

Reply:
Approving but please check this is not raising an error :)

Reply:
You are using `zone` so why rename it here?
But you are not using `stdout`, and as explained previously, as it's a named parameter, you cannot really rename it if it's a method called for many providers. The only possibility here is to add the linter ignore as you had previously

Reply:
Thanks for the quick comment!